code/__DEFINES/jobs.dm

```diff
@@ -21,6 +21,7 @@
 #define CHEMIST			(1<<2)
 #define CMO_JF			(1<<3)
 #define DOCTOR			(1<<4)
+#define EMT				(1<<2)

```

Comment:
🤔

Reply:
don't see what's wrong with this

Reply:
number order

Reply:
agh I thought this was jobs.txt for whatever reason, fixing it